Sources/NIOHTTP1/HTTPPipelineSetup.swift

```diff
@@ -28,9 +37,37 @@ extension ChannelPipeline {
     /// - returns: An `EventLoopFuture` that will fire when the pipeline is configured.
     public func addHTTPClientHandlers(position: Position = .last,
                                       leftOverBytesStrategy: RemoveAfterUpgradeStrategy = .dropBytes) -> EventLoopFuture<Void> {
-        return self.addHandlers(HTTPRequestEncoder(),
-                                ByteToMessageHandler(HTTPResponseDecoder(leftOverBytesStrategy: leftOverBytesStrategy)),
-                                position: position)
+        return self.addHTTPClientHandlers(position: position,
+                                          leftOverBytesStrategy: leftOverBytesStrategy,
+                                          withClientUpgrade: nil)
+    }
+    /// Configure a `ChannelPipeline` for use as a HTTP client with a client upgrader configuration.
+    ///
+    /// - parameters:
+    ///     - position: The position in the `ChannelPipeline` where to add the HTTP client handlers. Defaults to `.last`.
+    ///     - upgrade: Whether to add a `HTTPClientUpgradeHandler` to the pipeline, configured for
+    ///         HTTP upgrade. Defaults to `nil`, which will not add the handler to the pipeline. If
```

Comment:
This comment needs an update, as the `upgrade` parameter does not default to `nil`.

Reply:
Aaaaah, good spot. Done.